nml/actions/action0.py

```diff
@@ -205,12 +205,12 @@ def find_unused(self, length):
     BlockAllocation(0, -1, "Sound"),
     BlockAllocation(0, 127, "Airport"),
     BlockAllocation(0, -1, "Signal", False),
-    BlockAllocation(0, 255, "Object"),
+    BlockAllocation(0, 0xF9FF, "Object"), # 64000
```

Comment:
```suggestion
    BlockAllocation(0, 0xF9FF, "Object"),  # 64000
```

Reply:
In fact just s/0xF9FF/64000/ and remove the comment would do.

Reply:
Updated the suggestion